cli/formatter/table/README.md

```diff
@@ -0,0 +1,5 @@
+# formatter/table
+
+Based on github.com/jedib0t/go-pretty
```

Comment:
`formatter/table`&`formatter/text`:

After discussion with @LeonidVas and @psergee we decided that we need to do it in a different way:

1. Create a patch for the latest release of the `go-pretty`. The patch should be as small as possible.
2. Add `go-pretty` as a git submodule.
3. Patch the submodule with the patch on the build step.

See how it is already done with cartridge-cli in the repo.

Reply:
Done, implemented by analogy with the `cartridge-cli`